### DISCUSSION TITLE
The HTTP request failed to monitor the link

### DISCUSSION THREAD

**Discussion by rayejun** [apache/skywalking#7993] [Q&A]
Calling Dubbo service in spring boot HandlerInterceptor can only monitor the link of service invocation, and can not monitor the link of Http request.

Removing Dubbo calls from HandlerInterceptor can monitor the link of Http requests.

**Comment by wu-sheng** [comment]
HandlerInterceptor has an instrumentation target, you could check whether it is called in your case. If not, it means you need some change on Dubbo setting or have to enhance this plugin.

**Reply by rayejun** [reply]
What do you mean? I don't quite understand

**Reply by wu-sheng** [reply]
We are monitoring this filter. 

https://github.com/apache/skywalking-java/blob/fbdfc42b4d825ba33205d646ddaaaad20c005cb8/apm-sniffer/apm-sdk-plugin/dubbo-2.7.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/asf/dubbo/DubboInstrumentation.java?_pjax=%23js-repo-pjax-container%3Afirst-of-type%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%3Afirst-of-type%2C%20%5Bdata-pjax-container%5D%3Afirst-of-type#L33

I am not sure what is the status in your Dubbo setup. AFAIK, this filter could be disabled.

And check your Dubbo version whether in our supported framework document.
